scripts/tools/GlobalTimestamps.yml

```diff
@@ -37,22 +54,14 @@ params:
       name: hMetricGroup
       desc: "[in] handle of the metric group"
     - type: "$x_bool_t"
-      name: mappedToHost
-      desc: "[in] Are timestamps and metrics mapped to host, or device."
+      name: synchronizedWithHost
+      desc: "[in] Return host synchronized or device synchronized metric timestamp."
```

Comment:
Can we mention "Returns synchronized Host (or device) and metric timestamps."

This would be similar to the description for "zeDeviceGetGlobalTimestamps", which mentions "Returns synchronized Host and device global timestamps."

Reply:
@joshuaranjan here we are not returning both, only host or device sync'd.

Reply:
@jandres742 , Ok.

>Return host synchronized or device synchronized metric timestamp

This seemed to say that there is only one output value from this API i.e the "metric timestamp"

>"Returns synchronized Host (or device) and metric timestamps.

This seemed to say that there are two output values from this API 
1. the "metric timestamp"
2. Device OR host timestamp

May be this is subjective to my perception. Kiindly use which is better.
Thanks